data/result.go

```diff
@@ -18,19 +20,25 @@ import (
 type NDTResult struct {
 	// GitShortCommit is the Git commit (short form) of the running server code.
 	GitShortCommit string
+	// Version is the symbolic version (if any) of the running server code.
+	Version string
 
 	// All data members should all be self-describing. In the event of confusion,
 	// rename them to add clarity rather than adding a comment.
-	NDT5Metadata *ndt5data.Metadata `json:",omitempty"`
-
 	ServerIP   string
 	ServerPort int
 	ClientIP   string
 	ClientPort int
```

Comment:
By reading the code as-is, I'd assume these four describe the control channel for `ndt5` and the control+test channel for `ndt7`. Is my understanding correct?